tests/Functional/OrderUpdateTest.php

```diff
@@ -361,12 +395,12 @@ private function getEntityManager(): EntityManagerInterface
         return self::getContainer()->get('doctrine.orm.entity_manager');
     }
 
-    private function getInitialTotal(Order|InitialTotalAwareOrderInterface $order)
+    private function getInitialTotal(Order&EditableOrderInterface $order)
```

Comment:
I wonder, maybe we can use an `\Setono\SyliusOrderEditPlugin\Tests\Application\Entity\Order` here as a type hint? 🤔 we're in a test context, so we operate on the test application